provisioning/templates-api/wait.sh

```diff
@@ -18,7 +18,7 @@ if kubectl rollout status sts/templates-api-etcd --namespace templates-api --tim
 else
   echo
   echo "Etcd deployment failed."
-  minikube kubectl -- logs --namespace templates-api --follow=false --timestamps=true --selector "app=templates-api-etcd" &
+  minikube kubectl -- logs --namespace templates-api --follow=false --timestamps=true --selector "app=templates-api-etcd"
```

Comment:
Typo, logy sa maju vypisat, ale nie bezat ako background proces.
Pred tym som to mal vymyslene inak a zabudol som zmazat `&`.